api/v1alpha1/contentlibraryitem_types.go

```diff
@@ -74,6 +74,23 @@ type FileInfo struct {
 	Cached bool `json:"cached"`
 }
 
+// StorageInfo includes the storage backing for a file in a content library item
+// in vCenter.
+type StorageInfo struct {
+	// +required
+
+	// Name specifies the name of the file in vCenter.
+	Name string `json:"name"`
+
+	// +required
+
+	// StorageURIs identify the file on the storage backing. These URIs may be
+	// specific to the backing and may need interpretation by the client. A
+	// client that understands a URI scheme in this list may use that URI to
+	// directly access the file on the storage backing.
+	StorageURIs []string `json:"storageURIs"`
```

Comment:
Do these URIs point to iso files? I wanna understand why naming `StorageInfo`.

Reply:
Yes, it's used when we create a CD-ROM to specify its backing file to mount ISO. I moved it to the `FileInfo` field as Eric suggested below and added more comments to clarify the usage.